client/cmd_test.go

```diff
@@ -1,3 +1,4 @@
+// nolint: misspell
```

Comment:
huh? Why don't we just fix the misspelling?

Reply:
Because the text is about testing the misspell lol